### PR TITLE
Project scaffolding, build/dev process, db init, placeholder bot. Closes #1.

### LINKED ISSUE #1: Create basic bot repo from pombot.

### PR DESCRIPTION
**Pull request by cowboy**

### CODE REVIEW

**Comment by cowboy** [comment]
I still need to init the experience and interest tables.

**Comment by cowboy** [comment]
@tkellen can I get a review of this? THX U

**Comment by tkellen** [comment]
coming up!

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+console.log(`Server running on ${config.app.host}:${config.app.port}.`);
+
+// =============================
+// Start and stop bots as needed
+// =============================
+
+const bots = {};
+
+// Start a bot if it's not already running.
+function startBot(token) {
```
Comment (tkellen) [inline src/app.js]: feels like this belongs under `src/services/`

**Comment by tkellen** [comment]
:+1: other than that minor nit

**Comment by cowboy** [comment]
@tkellen any more feedback before i merge?

**Comment by tkellen** [comment]
negative, carry on!

**Pull request merged**